src/webapp/app.py

```diff
@@ -348,3 +363,5 @@ def logout():
         return redirect(url_for("index"))
 
     return app
+
+ # <ms_docref_logout_user>
```

Comment:
Typo, also the indent doesn't match the indent of the opening tag.  And they _both_ have extra newlines (only one fixed here, but you can fix the other).

```suggestion
    return ap
    # </ms_docref_logout_user>
```